resources/gnomad_liftover.py

```diff
@@ -0,0 +1,160 @@
+import argparse
+from basics import *
+from gnomad_hail import *
+import logging
+
+
+logging.basicConfig(format="%(levelname)s (%(name)s %(lineno)s): %(message)s")
+logger = logging.getLogger("gnomAD_liftover")
+logger.setLevel(logging.INFO)
+
+
+def checkpoint(ht: hl.Table, dtype: str) -> hl.Table:
+    """
+    Creates a "checkpoint" by writing Table out and reading back in
+    :param Table ht: Table to be evaluated and saved
+    :param str dtype: Data type (exomes or genomes)
+    :return: Table after writing out and reading back in
+    :rtype: Table
+    """
+
+    if dtype == 'exome':
+        ht.write('gs://seqr-datasets/methods_dev/test_data/kc/gnomAD_exome_sites.ht', overwrite=True)
```

Comment:
Should probably move these paths to the top of the script. You could do something like:
```
intermediate_liftover_path = 'gs://seqr-datasets/methods_dev/test_data/kc/gnomAD_{}_sites.ht'
```
And then when you go to use it:
```
intermediate_liftover_path.format(data_type)
```